client/components/shipping/packages/add-package.js

```diff
@@ -79,7 +87,6 @@ const useDefaultField = ( value, updatePackagesField ) => {
 		is_letter: 'envelope' === value,
 		name: null,
 		is_user_defined: true,
-		outer_dimensions: null,
```

Comment:
`null` values here are pruned in the reducer - I put it there as a mechanism to delete object properties by way of an action. See: https://github.com/Automattic/woocommerce-connect-client/blob/master/client/state/form/packages/reducer.js#L52